run.py

```diff
@@ -93,14 +97,19 @@ def cleanup():
 def main():
     args = get_args()
     yaml_path = os.path.join(os.getcwd(), args.configfile)
-    # container name
-    container = "jupyter-" + args.session
     tasks = get_config(yaml_path)
+    # These parameters are needed by 'TestBase.py' file for output conf
+    #TODO
```

Comment:
Is this the final version?
I am not too much in favor of writing a pickle file to pass args...